realm/src/androidTest/java/io/realm/RealmTest.java

```diff
@@ -2013,12 +2013,13 @@ public void run() {
         }
     }
 
-    public void testProcessLocalListenersAfterRefresh() {
+    public void testProcessLocalListenersAfterRefresh() throws InterruptedException {
```

Comment:
You mean throws? Any exception not caught will fail the unit test anyway, so we need only to declare any checked exceptions?